<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1581</width>
     <height>881</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="g1">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="sourceGrid">
      <item>
       <widget class="QComboBox" name="sourceCB"/>
      </item>
      <item>
       <widget class="QComboBox" name="fileCB"/>
      </item>
      <item>
       <widget class="QSpinBox" name="cameraIdSB"/>
      </item>
      <item>
       <widget class="QPushButton" name="cameraButton">
        <property name="text">
         <string>Start capture</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1000</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="g2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="imageGrid">
        <item row="0" column="2">
         <widget class="QSlider" name="ROI_y2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QGridLayout" name="ROIGrid">
          <item row="0" column="3">
           <widget class="QLabel" name="ROISizeLabel">
            <property name="text">
             <string>ROISizeLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="showROIChkBox">
            <property name="text">
             <string>Show ROI</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="rawSizeLabel">
            <property name="text">
             <string>rawSizeLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="ROI_y1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="rawImageDisplayLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>rawImageDisplayLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="ROI_x2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="ROI_x1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="ClickableLabel" name="processedImageDisplayLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>processedImageDisplayLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>400</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="g3">
        <item row="0" column="0">
         <widget class="QLabel" name="profileImageLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>profileImageLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="g4">
          <item row="2" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="controlGrid" columnminimumwidth="100,50">
            <item row="1" column="0">
             <widget class="QLabel" name="processingTypeLabel">
              <property name="text">
               <string>Processing method</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="rotationLabel">
              <property name="text">
               <string>Rotation</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="rotationCB"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="channelAmplifyLabel">
              <property name="text">
               <string>Amplify channel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="channelIsolateLabel">
              <property name="text">
               <string>Isolate channel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="controlGridLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Controls</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="channelIsolateCB"/>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="edgeExtractCB"/>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="processingTypeCB"/>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="processROIChkBox">
              <property name="text">
               <string>Process ROI only</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="channelAmplifyCB"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="edgeExtractLabel">
              <property name="text">
               <string>Edge extraction method</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="paramGrid">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="3" column="2">
             <widget class="QLabel" name="cannyRatioValueLabel">
              <property name="text">
               <string>cannyRatioValueLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="cannyThrLabel">
              <property name="text">
               <string>Canny threshold</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="gaussKernelLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Gauss kernel size</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="cannyThrSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="regionThrLabel">
              <property name="text">
               <string>Region growing threshold</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSlider" name="regionThrSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="gaussKernelSB">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="cannyKernelLabel">
              <property name="text">
               <string>Canny kernel size</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="regionThrValueLabel">
              <property name="text">
               <string>regionThrValueLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="paramGridLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Processing parameters</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="regionChkBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Region growing active</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="cannyRatioSlider">
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="seedLabel">
              <property name="text">
               <string>Seed point: </string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="cannyThrValueLabel">
              <property name="text">
               <string>cannyThrValueLabel</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="cannyRatioLabel">
              <property name="text">
               <string>Canny ratio</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="cannyKernelSB">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>7</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="geometryGrid">
            <item row="5" column="0">
             <layout class="QGridLayout" name="gridLayout"/>
            </item>
            <item row="4" column="0">
             <widget class="QSlider" name="cameraElevationSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QSlider" name="FOVxSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QSlider" name="viewAngleSlider">
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="geometryGridLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Camera and geometry</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="viewAngleLabel">
              <property name="text">
               <string>View angle</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="cameraElevationLabel">
              <property name="text">
               <string>Camera elevation</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QSlider" name="FOVySlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <layout class="QGridLayout" name="pixelCntGrid">
              <item row="0" column="0">
               <widget class="QLabel" name="pixelCntLabel">
                <property name="text">
                 <string>Pixel count</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QCheckBox" name="pixelCntFromImageChkBox">
                <property name="text">
                 <string>From image</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="pixelCntXSB">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="xLabel">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="pixelCntYSB">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="FOVLabel">
              <property name="text">
               <string>FOV</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>383</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
